mindboggle/mindboggle123

```diff
@@ -87,6 +87,9 @@ adv_args.add_argument("--fs_openmp", dest="openmp",
 adv_args.add_argument("--fs_T2image", dest="T2image",
                       type=str,
                       help="Optional T2 image to use with FreeSurfer")
+adv_args.add_argument("--fs_3T", dest="T3",
```

Comment:
Hey @satra, thanks for checking!

do you mean something in the line of this: 

    adv_args.add_argument("--fs_flags", dest="fs_flags",
                             type=str,
                             help="include additional recon-all flags like e.g. '-nuintensitycor-
                             3T' ")
    if args.fs_flags:
           reconall.inputs.flags = args.fs_flags

where args.fs_flags is a list of flags defined by the user?